Updates to Add-ons section of Quick start guide (#16618)
nvaccess/nvda: user_docs/en/userGuide.md

#### Available add-ons {#AvailableAddons}
When the window first opens, add-ons may take a few seconds to load.
NVDA will read the name of the first add-on once the list of add-ons finishes loading.

> Available add-ons are listed alphabetically in a multi-column list.

Author: Better use the word "list" everywhere rather than "list" once, "grid" once and "table" another time.

To browse the list and find out about a specific add-on:

> 1. Use the arrow keys or press the first letter of an add-on name to move around the list.

Author: Only format the name of a key.

1. Press `tab` once to move to a description of the currently selected add-on.
1. Use the [reading keys](#ReadingText) or arrow keys to read the full description.
1. Press `tab` to the "Actions" button, which can be used to install the add-on, among other actions.
1. Press `tab` to "Other Details", which lists details such as the publisher, version and homepage.

> 1. To return to the list of add-ons, press `alt+a`, or multiple times `shift+tab`.

Author: * Avoids a strange repetition
* "shift+tab back" seems a pleonasm to me: either you press shift+tab or you tab back.

Reviewer: ```suggestion
1. To return to the list of add-ons, press `alt+a`, or `shift+tab` multiple times.
```

```suggestion
1. To return to the list of add-ons, press `alt+a`, or `shift+tab` until you reach it.
```

Reviewer 2: ```suggestion
1. To return to the list of add-ons, press `alt+a`, or `shift+tab` until reaching the list.

```

#### Searching for add-ons {#SearchingForAddons}
As well as browsing all available add-ons, it is possible to filter the add-ons shown.

> To search, press `alt+s` to jump to the "Search" field and type the text to search for.

Author: The label "Search" is capitalized in the GUI, so it should also be when you quote it here.

Reviewer 2: Happy for the change of "search" to "Search", but did you intend to remove the next two lines?

Reviewer: @Qchristensen - they were modified, the equivalent changes I've discussed here: https://github.com/nvaccess/nvda/pull/16618#discussion_r1616431530

> The content of the search field is searched in the add-on ID, display name, publisher, author and description.

Reviewer: The phrasing here is quite awkward

```suggestion
Searching checks for matches in the add-on ID, display name, publisher, author and description fields.
```

1. With the focus on an add-on you would like to install, press `enter`.
1. The actions menu opens with a list of actions; the first action is "Install".

> 1. To install the add-on, press `i` or `downArrow` to "Install" and press `enter`.

Author: Idem: label capitalized in GUI, so also here.

1. The focus returns to the add-on in the list and NVDA will read the details about the add-on.
1. The "Status" information reported by NVDA changes from "Available" to "Downloading".
1. Once the add-on has finished downloading, it will change to "Downloaded. Pending install".
1. Repeat with any other add-ons you would like to install at the same time.
1. Once finished, press `tab` until the focus is on the "Close" button, then press `enter`.
1. The downloaded add-ons will start the installation process once the Add-on Store is closed.
During the installation process, add-ons may display dialogs that you will need to respond to.
1. When the add-ons have been installed, a dialog appears advising that changes were made, and you must restart NVDA for the add-on installation to complete.
1. Press `enter` to restart NVDA.

#### Managing installed add-ons {#ManagingInstalledAddons}

> Press `control+tab` to move between the tabs of the Add-on Store.

Author: What are the sections of the Add-on Store? If it's a synonym of "tab", it's not needed IMO.

The tabs include: "Installed add-ons", "Updatable add-ons", "Available add-ons" and "Installed incompatible add-ons".

> Each of the tabs are set out similar to each other, as a list of add-ons, a panel for more details on the selected add-on, and a button to perform actions for the selected add-on.

Author: "the ability" is not a part of the tab.

The actions menu of installed add-ons includes "Disable" and "Remove" rather than "Install".
Disabling an add-on stops NVDA from loading it, but leaves it installed.
To re-enable a disabled add-on, activate "Enable" from the actions menu.
After enabling, disabling, or removing add-ons, you will be prompted to restart NVDA when closing the Add-on Store.
These changes will only take effect once NVDA is restarted.

> Note that in the Add-on Store window `escape` works the same as the Close button.

Author: It was not very clear if "dialog" refers to the restart prompt or to the add-on store window.

#### Updating add-ons {#UpdatingAddons}

> When an update to an add-on you have installed is available, it will be listed in the "Updatable add-ons" tab.

Author: Better use the word "list" everywhere rather than "list" once, "grid" once and "table" another time.
